lib/pure/strutils.nim

```diff
@@ -324,6 +324,18 @@ proc toOctal*(c: char): string {.noSideEffect, rtl, extern: "nsuToOctal".} =
     result[i] = chr(val mod 8 + ord('0'))
     val = val div 8
 
+proc isEmpty*(s: string): bool {.noSideEffect, procvar, rtl, extern: "nsuIsEmpty".}=
+  ## Checks whether the string `s` is empty or not.
+  ##
+  ## In this function whitespaces are stripped from `s`
+  if s.len() == 0:
+    return true
+
+  if s == nil or s == "" or s.strip() == "":
```

Comment:
I agree that it should have some better name. ``isPresent`` doesn't mean anything to me though, but since Python has nothing like it we can only copy Ruby's names, I guess.